scripts/lib/mappings.json

```diff
@@ -0,0 +1,23 @@
+{
+  "options": {
+    "CONFIG_MAINBOARD_POWER_FAILURE_STATE": "DEFAULT_POWER_STATE_AFTER_FAIL",
+    "CONFIG_VBOOT": "VERIFIED_BOOT_SUPPORT",
+    "CONFIG_TPM": "TPM_SUPPORT",
+    "CONFIG_EDK2_SECURE_BOOT_SUPPORT": "SECURE_BOOT_SUPPORT",
+    "CONFIG_EDK2_IPXE_OPTION_NAME": "IPXE_BOOT_ENTRY",
+    "CONFIG_TPM_MEASURED_BOOT": "SERIAL_BOOT_MEASURE",
```

Comment:
I'm pretty sure this one is just about measuring boot time with cbmem, nothing to do with TPM

Reply:
Right, [removed](https://github.com/Dasharo/open-source-firmware-validation/pull/392/commits/3c56ce6b6115def3dc0cd03478686b2c56445274#diff-63d9f43eb53de310c84792f587f9e3d6d867882fe3f7a4ec78e13dabf541a488L8)